.github/workflows/main.yml

```diff
@@ -13,7 +13,21 @@ on:
       - "tests/**"
 
 jobs:
+  test-lint:
+    name: lint-python${{ matrix.python-version }}
+    strategy:
+      matrix:
+        python-version:
+         - "3.6"
+         - "3.10"
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: psf/black@stable
+
   test-sqlite:
+    needs: 
```

Comment:
Chaining the tasks seems to increase CI time
So, can we run lint in parallel with the other tests
BUT can we configure github-actions to STOP all jobs if any 1 job fails.
i.e. if lint fails in 2mins (because of an extra space), stop all the tests